Library/Homebrew/cmd/--cache.rb

```diff
@@ -2,24 +2,31 @@
 
 require "fetch"
 require "cli/parser"
+require "cask/cmd"
+require "cask/cask_loader"
 
 module Homebrew
   module_function
 
   def __cache_args
     Homebrew::CLI::Parser.new do
       usage_banner <<~EOS
-        `--cache` [<options>] [<formula>]
+        `--cache` [<options>] [<formula/cask>]
```

Comment:
```suggestion
        `--cache` [<options>] [<formula>]
```
The slash here interferes with the man page and Markdown generation. Just "formula" is fine, since a formula file is used for both formulae and casks.